src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -623,6 +642,56 @@ def __print_restore_preview(kvs_to_restore, kvs_to_modify, kvs_to_delete):
     return True
 
 
+def __is_json_content_type(content_type):
+    # pylint: disable=too-many-return-statements
+    if not content_type:
+        return False
+
+    content_type = content_type.strip().lower()
+    mime_type = content_type.split(';')[0].strip() if ';' in content_type else content_type
+    type_parts = mime_type.split('/') if '/' in mime_type else None
+
+    if not type_parts or len(type_parts) != 2:
+        return False
+
+    (main_type, sub_type) = type_parts
+    if main_type != "application":
+        return False
+    if "json" not in sub_type:
+        return False
```

Comment:
no need to check here. Even "json" in sub_type it can be application/xxxjsonxxx which is still not json.